tests/c/build.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
```

Comment:
just looking at the https://github.com/WebAssembly/wasi-testsuite/pull/114/files and it seems a lot of duplication, I wonder if we can refactor it in a way that the common part can be reused across different languages?

Reply:
I think yes, but wdyt about me doing that in a followup?  It's a bit easier when the three implementations are actually there in the source tree.